packages/fx-core/resource/package.nls.json

```diff
@@ -440,8 +440,10 @@
   "core.createProjectQuestion.ApiKeyConfirm": "Microsoft 365 Agents Toolkit will upload the API key to Developer Portal. The API key will be used by Teams client to securely access your API in runtime. Microsoft 365 Agents Toolkit will not store your API key.",
   "core.createProjectQuestion.OauthClientId": "Enter client id for OAuth registration in OpenAPI Description Document",
```

Comment:
Title: "OAuth registration client ID"
Tips in input box: "Input client ID for OAuth registration in OpenAPI Description Document"